tools/spm/templates/COMPONENT_SPE/TARGET_MBED_SPM/psa_NAME_partition.c.tpl

```diff
@@ -23,6 +23,8 @@
  * Generated by tools/spm/generate_partition_code.py Version {{script_ver}}
  **********************************************************************************************************************/
 
+#if !defined(TARGET_TFM)
```

Comment:
Not needed if the location of the generated file is inside `TARGET_MBED_SPM` folder.